common-lib/src/main/scala/com/gu/mediaservice/lib/config/MetadataStore.scala

```diff
@@ -0,0 +1,54 @@
+package com.gu.mediaservice.lib.config
+
+import com.gu.mediaservice.lib.BaseStore
+import org.joda.time.DateTime
+import play.api.Logger
+import play.api.libs.json.Json
+
+import scala.concurrent.ExecutionContext
+import scala.util.{Failure, Success, Try}
+
+class MetadataStore(bucket: String, config: CommonConfig)(implicit ec: ExecutionContext)
```

Comment:
Doesn't look like `config` is being used in this class, so we can remove?

Reply:
Looks like this is required by `BaseStore` below (in order to build the s3 client). Similar is being done in `KeyStore.scala`, maybe there is a nicer way of doing this?

Reply:
Looks like this is required by `BaseStore` below (in order to build the s3 client). Similar is being done in `KeyStore.scala`, maybe there is a nicer way of doing this?